src/_utils/ApiWorkaroundUtils.js

```diff
@@ -0,0 +1,16 @@
+import * as LiveData from '../_data/LiveData';
+
+export const forceStatementUpdate = () => {
+    const todayEpoch = Math.floor(Date.now() / 1000) - 86400;
```

Comment:
Not so sure, as it's difficult to name, it actually means the epoch on the starting point of today ~ 
Should I?